editor/src/com/talosvfx/talos/editor/addons/scene/widgets/gizmos/PaintSurfaceGizmo.java

```diff
@@ -148,26 +149,37 @@ public void touchDown(float x, float y, int button) {
             }
         }
 
-        if(frameBuffer == null || sizeIsDifferent) {
-            createFrameBuffer();
+        if (frameBuffer == null || sizeIsDifferent) {
+           FrameBuffer fbo = createFrameBuffer();
+           if (fbo == null) {
```

Comment:
when is this ever the case?

Reply:
when painable surface is created and user clicks on it, it attempts to create frame buffer, while texture is not selected. Because, it doesn't have texture, frame buffer creation fails from null pointer expectation, so we try to skip it.